content/docs/quickstart/kotlin.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: quickstart
+title: Kotlin Quick Start
+short: Kotlin
+description: This guide gets you started with gRPC in Kotlin with a simple working example.
+---
+
+<div id="toc"></div>
+
+### Prerequisites
+
+- Kotlin version 1.3 or higher
```

Comment:
@lowasser @bshaffer is this reasonable?
Any other prerequisites?

Reply:
1.3+ as confirmed in https://github.com/grpc/grpc.io/issues/132#issuecomment-600210551